### DISCUSSION TITLE
Location detection

### DISCUSSION THREAD

**Discussion by vladmihociu** [teslamate-org/teslamate#3453] [Ideas]
Hello everyone,

I would like to create an "automatization" in the Grafana, that will trigger when I am arriving home with my Tesla. I would like to turn on a few exterior and interior lights by accessing a link. 

Is this possible? 

Thanks!

**Comment by pborgdk** [comment]
I don't specifically know Grafana - but Home Assistant or Nodered would probably be good suggestions for automation.

**Comment by zlymeda** [comment]
I don't think that is possible in grafana. you might consider home assistant for this

**Comment by DrMichael** [comment]
Actually anthing using MQTT would help here. Grafana does not.
